spring-cloud-function-integration/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-function-integration</artifactId>
+	<name>Spring Cloud Function with Spring Integration</name>
+	<description>Spring Cloud Function with Spring Integration</description>
+
+	<parent>
+		<artifactId>spring-cloud-function-parent</artifactId>
+		<groupId>org.springframework.cloud</groupId>
+		<version>4.0.3-SNAPSHOT</version>
+	</parent>
+
```

Comment:
The parent pom brings us `<java.version>17</java.version>`.
I don't see a reason in anything else.
Plus that `.jdk8` is probably a bit outdated since the whole project is Java 17 already :shrug: